doc/classes/ScriptExtension.xml

```diff
@@ -150,7 +150,7 @@
 		<method name="_is_abstract" qualifiers="virtual const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the script is an abstract script. An abstract script does not have a constructor and cannot be instantiated.
+				Returns [code]true[/code] if the script is an abstract script. Abstract scripts are not intended to be used directly, instead other scripts should inherit them. An abstract script does not have a constructor and cannot be instantiated by itself. They will be either unselectable or hidden in the Create New Node dialog (unselectable if there are non-abstract classes inheriting it, otherwise hidden).
```

Comment:
```suggestion
				Returns [code]true[/code] if the script is an abstract script. Abstract scripts are not intended to be used directly, instead other scripts should inherit them. An abstract script does not have a constructor and cannot be instantiated. They will be either unselectable or hidden in the Create New Node dialog (unselectable if there are non-abstract classes inheriting it, otherwise hidden).
```

What do you have in mind? I think this is confusing. Is there some way to instantiate an abstract class?

Reply:
By extending it.

Reply:
```suggestion
				Returns [code]true[/code] if the script is an abstract script. Abstract scripts cannot be instantiated directly, instead other scripts should inherit them. Abstract scripts will be either unselectable or hidden in the Create New Node dialog (unselectable if there are non-abstract classes inheriting it, otherwise hidden).
```